frontend/src/views/SettingsView/components/ConnectedApplications.vue

```diff
@@ -57,6 +59,7 @@ const initialCalendars = computed(() => {
       type_id: externalConnection?.type_id,
       connection_name: externalConnection?.name,
       provider_name: connectionProvider,
+      is_default: currentState.value.defaultCalendarId === calendar.id,
```

Comment:
This improves the markup a lot, because we now already have that information as property